portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles user authentication. */
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+  
+  private static final String REDIRECT_URL = "/blog.html";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
```

Comment:
Not for this PR since its more of a `refactoring` or cleaner code kind of change, but -- 

I've just realized that with the usage of servlets, this is going to be something that keeps on appearing.

High-level, the content type (which I think is restricted to be MimeType). With future different types o f conetnts (images-jpg/png, html, plain text, etc) -- [MIME types list](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types), it might be good to store a subset of those in some data strcuture outside of the servlet and refer to those in your servlets.

I've just realized that with the usage of servlets, this is going to be something that keeps on appearing.

High-level, the content type (which I think is restricted to be MimeType). With future different types o f contents (images-jpg/png, html, plain text, etc) -- [MIME types list](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types), it might be good to store a subset of those separately and then refer to them from here.